components/list/list-item.js

```diff
@@ -26,65 +29,104 @@ class ListItem extends RtlMixin(LitElement) {
 		const layout = css`
 			:host {
 				display: block;
-				margin: 1px 0;
+				margin: 0.05rem 0;
 			}
 			:host[hidden] {
 				display: none;
 			}
 			.d2l-list-item-flex {
 				display: flex;
+				position: relative;
 			}
 			.d2l-list-item-content {
+				background: var(--d2l-list-item-background, none);
 				border-bottom: var(--d2l-list-item-divider-bottom, 1px solid var(--d2l-color-mica));
 				border-top: var(--d2l-list-item-divider-top, 1px solid var(--d2l-color-mica));
 				box-sizing: content-box;
-				margin-bottom: -1px;
-				margin-top: -1px;
+				margin-bottom: -0.05rem;
+				margin-top: -0.05rem;
 				padding-bottom: var(--d2l-list-item-divider-padding-bottom, 0);
 				padding-top: var(--d2l-list-item-divider-padding-top, 0);
 				position: relative;
 				width: 100%;
 			}
+			.d2l-list-item-flex:hover .d2l-list-item-content {
+				background: var(--d2l-list-item-hover-background, none);
+			}
 			.d2l-list-item-content-flex {
 				display: flex;
 				flex-grow: 1;
 				justify-content: stretch;
-				margin: 18px 0;
-				padding: var(--d2l-list-item-content-padding, 0);
+				padding: 0.55rem var(--d2l-list-item-content-padding-side, 0);
 			}
-			::slotted([slot="illustration"]){
-				align-self: flex-start;
+			::slotted([slot="illustration"]) {
 				display: flex;
 				flex-grow: 0;
 				flex-shrink: 0;
+				margin: 0.15rem 0;
 				margin-right: 0.9rem;
-				max-height: 52px;
-				max-width: 90px;
+				max-height: 2.6rem;
+				max-width: 4.5rem;
 				overflow:hidden;
 			}
 			:host([dir="rtl"]) ::slotted([slot="illustration"]) {
 				margin-left: 0.9rem;
 				margin-right: 0;
 			}
-			:host([illustration-outside]) ::slotted([slot="illustration"]) {
-				margin-bottom: 18px;
-				margin-top: 18px;
-			}
 			::slotted([slot="actions"]) {
 				align-self: flex-start;
-				display: flex;
+				display: grid;
 				flex-grow: 0;
+				grid-auto-columns: 1fr;
+				grid-auto-flow: column;
+				grid-gap: 0.3rem;
+				margin: 0.15rem 0;
+				position: relative;
+				z-index: 150;
```

Comment:
How was `150` determined as the right z-index value?

Changing positioning and `z-index` of the `::slotted` items worries me.  When possible, I think I'd rather do as much locally as possible and not apply styles to user-land DOM elements.

Reply:
So why 150 - I was thinking I could make all z-index I use for this component to live in the 100s. I'm happy with best long term solution so changing to meet that I am all for. (Note, I need to double check that all my z-index actually fitting to what I just said)

So to tell you the problem I am solving with the z-index:
So clicking on the list item anywhere should click the checkbox. Unless you are directly on a secondary action. So I needed to put it above the label.

If someone has two actions then the gap between should also click the checkbox.